### PR TITLE
debian: rename binary/source to snapd

### PR DESCRIPTION
**Pull request by mvo5**
This renames the ubuntu-snappy package to snapd. It also renames some of the services files. Please let me know if my approach of using "snapd" here makes sense or if you would prefer something else.

### CODE REVIEW

**Comment by mvo5** [comment]
retest this please

File: README.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ There is more to read about the testing framework on the [website](https://labix
 To test the `snapd` REST API daemon on a snappy system you need to
 transfer it to the snappy system and then run:
 
-    sudo systemctl stop ubuntu-snappy.snapd.service ubuntu-snappy.snapd.socket
+    sudo systemctl stop snapd.snapd.service snapd.snapd.socket
```
Comment (chipaca) [inline README.md]: ...

Just make it `snapd.socket`?

File: debian/snapd.dirs [hunk 2]
```diff
@@ -1,2 +1,4 @@
 /snaps
 /var/lib/snappy/apparmor/additional
+var/lib/snappy/snaps
+usr/lib/snappy
```
Comment (chipaca) [inline debian/snapd.dirs]: I think you need a newline there

**Comment by chipaca** [comment]
I really don't like that all the services that are about ubuntu snappy are now called `snapd.something`. I can see why we're doing it, but I don't like it :-)

Other than that, and a couple of ignorable nits, :+1:

**Comment by zyga** [comment]
+1 though I wonder if you agree with chipaca's suggestion to just call it snapd.{socket,service}.
Please resolve merge conflicts.

File: README.md (hunk 1 above)
Comment (mvo5) [inline README.md]: Good idea!

**Comment by mvo5** [comment]
Thanks @zyga and @chipaca ! I renamed snapd.snapd.snapd.snapd.{socket,service} to just snapd.{socket,service} now, I think that makes sense. I also don't like the whole snapd.* thing.  I wonder if we should just call it snap.* instead? The existing snap package does not use any services (and is unlikely to ever use them).

**Comment by niemeyer** [comment]
How about ubuntu-core.* instead of snap.*? Then it's our namespace.

LGTM either way.

**Comment by chipaca** [comment]
@mvo5 needs deconflicting

**Pull request merged**